crates/re_space_view_spatial/src/ui.rs

```diff
@@ -693,3 +694,10 @@ pub fn picking(
 
     Ok(response)
 }
+
+fn hit_ui(ui: &mut egui::Ui, hit: &crate::picking::PickingRayHit) {
+    if hit.hit_type == PickingHitType::GpuPickingResult {
+        let glam::Vec3 { x, y, z } = hit.space_position;
+        ui.label(format!("Picking position: [{x:.5}, {y:.5}, {z:.5})"));
```

Comment:
At some point we should figure out a way to unify all formatting of math stuff like this, behind something similar to the `DataUi` trait, but simpler

Reply:
```suggestion
        ui.label(format!("Picking position: [{x:.5}, {y:.5}, {z:.5}]"));